tests/coatils_test.py

```diff
@@ -94,10 +97,11 @@ def test_run_coala(self):
                       self.pop_message())
 
     def test_construct_settings(self):
-        self.assertEqual(Coatils.construct_settings('bear1 a=1 b=2 bear2 bear3'),
-                         {'bear1': {'a': '1', 'b': '2'},
-                          'bear2': {},
-                          'bear3': {}})
+        self.assertEqual(
+            Coatils.construct_settings('bear1 a=1 b=2 bear2 bear3'),
+            {'bear1': {'a': '1', 'b': '2'},
+             'bear2': {},
+             'bear3': {}, })
```

Comment:
cripes.
please read about trailing comma.
https://coderwall.com/p/4fvcaa/trailing-comma-in-python-data-structures
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas